torch/csrc/distributed/rpc/python_rpc_handler.h

```diff
@@ -7,16 +7,33 @@ namespace torch {
 namespace distributed {
 namespace rpc {
 
-namespace PythonRpcHandler {
-// initialize python module object and function objects in which python user
-// defined function (UDF) will run there
-void init();
-// execute python UDF, result is pickled to binary string
-std::vector<char> generatePythonUDFResult(const Message& request);
-// returned python UDF result is pickled binary string, so run python
-// function to unpickle the python UDF result and return pyObject to user
-py::object loadPythonUDFResult(const Message& message);
-} // namespace PythonRpcHandler
+// Singleton class provides interface to  execute python UDF remote call
+// and deserialize the returned results by running python function
+// in internal_rpc_utilities.
+// The singleton object is constructed at first when RPC agent is
+// constructed, where the python function in
+// torch/distributed/internal_rpc_utils.py are imported only once.
+class PYBIND11_EXPORT PythonRpcHandler {
```

Comment:
Question: what do we need `PYBIND11_EXPORT` here?

Reply:
otherwise, it will have warning/error "PythonRpc hander declared with greater visibility than the type of its field", this is know issue when including pybind11 types such as py::object https://fburl.com/0b0gobwx, adding this can fix this warning/error in OSS

Reply:
@gchanan @ezyang 

It seems this is the first time we use `PYBIND11_EXPORT`. Do we have our own version of `PYBIND11_EXPORT`, or is it OK to use it here?